python/lsst/sconsUtils/dependencies.py

```diff
@@ -504,7 +504,6 @@ def _tryImport(self, name):
                 else:
                     module.config.addCustomTests(self.customTests)
                 return module
-        state.log.info("Failed to import configuration for optional package '%s'." % name)
```

Comment:
It's still not obvious why the message is there and why it thinks it's only used for optional packages.